bundle/Form/Type/AbstractContentType.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\InformationCollectionBundle\Form\Type;
```

Comment:
Since we have this namespace how, move other form types here.

Reply:
solved in 49dc73ca